qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/AffiliationSelectionEvent.groovy

```diff
@@ -0,0 +1,26 @@
+package life.qbic.portal.qoffer2.web.views
```

Comment:
NIT: should events be part of the views package?

Reply:
Good question. Since it is only relevant in the view package I see no reason to move it elsewhere but I am open for suggestions :)

Reply:
Honestly I don't know either